subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/transform/DefaultTransformer.java

```diff
@@ -82,14 +90,30 @@
 
     private IsolatableParameters isolatable;
 
-    public DefaultTransformer(Class<? extends ArtifactTransformAction> implementationClass, @Nullable Object parameterObject, ImmutableAttributes fromAttributes, Class<? extends FileNormalizer> inputArtifactNormalizer, Class<? extends FileNormalizer> dependenciesNormalizer, ClassLoaderHierarchyHasher classLoaderHierarchyHasher, IsolatableFactory isolatableFactory, ValueSnapshotter valueSnapshotter, PropertyWalker parameterPropertyWalker, DomainObjectProjectStateHandler projectStateHandler, InstantiationScheme actionInstantiationScheme) {
+    public DefaultTransformer(
+        Class<? extends ArtifactTransformAction> implementationClass,
+        @Nullable Object parameterObject,
+        ImmutableAttributes fromAttributes,
+        Class<? extends FileNormalizer> inputArtifactNormalizer,
+        Class<? extends FileNormalizer> dependenciesNormalizer,
+        ClassLoaderHierarchyHasher classLoaderHierarchyHasher,
+        IsolatableFactory isolatableFactory,
+        ValueSnapshotter valueSnapshotter,
+        FileCollectionFactory fileCollectionFactory,
+        FileCollectionFingerprinterRegistry fileCollectionFingerprinterRegistry,
```

Comment:
Same here: why do we need these two in the transformer itself? Maybe this is another reason against in-place isolation, too?